addons/common/stringtable.xml

```diff
@@ -1828,5 +1828,20 @@
             <Chinese>下方</Chinese>
             <Turkish>Alt</Turkish>
         </Key>
+        <Key ID="STR_ACE_Common_QTEInput">
+            <English>Command</English>
+        </Key>
+        <Key ID="STR_ACE_Common_QTEAccepted">
+            <English>Accepted</English>  
+        </Key>
+        <Key ID="STR_ACE_Common_QTECancelled">
+            <English>Command Cancelled</English>  
+        </Key>
+        <Key ID="STR_ACE_Common_QTEHoldKey">
+            <English>Hold Down Key</English>  
+        </Key>
+        <Key ID="STR_ACE_Common_QTEInputKeys">
+            <English>Input Keys</English>  
+        </Key>
```

Comment:
What is our standard translate method? Google Translate?

Reply:
Individual contributors take care of it, we don't machine translate anything.